pallets/runtime/develop/src/fee_details.rs

```diff
@@ -56,15 +54,21 @@ impl CddAndFeeDetails<AccountId, Call> for CddHandler {
     fn get_valid_payer(
         call: &Call,
         caller: &Signatory<AccountId>,
```

Comment:
Do we need a Signatory type here?

Reply:
Agreed - will capture factoring this out in a separate JIRA.